aspnetcore/host-and-deploy/blazor/webassembly.md

```diff
@@ -394,3 +394,53 @@ A Blazor WebAssembly app can be initialized with the `loadBootResource` function
 External sources must return the required CORS headers for browsers to allow the cross-origin resource loading. CDNs usually provide the required headers by default.
 
 You only need to specify types for custom behaviors. Types not specified to `loadBootResource` are loaded by the framework per their default loading behaviors.
+
+## Rename DLL files to bypass proxies, firewalls, and antivirus software
+
+Some environments block the download of *.dll* files that are normally part of a Blazor app's published deployment. To address this issue, you can rename *.dll* files to use a different file extension and update the *blazor.boot.json* file.
```

Comment:
Rather than phrase it like this, I think we should use phrasing more like the following:

> Changing the filename extension of .dll files
>
> In case you have any reason to change the filename extensions for the `.dll` files in your application, you can follow the procedure below.

That is, I *don't* think we should imply this is something people should do "just in case". We only have a handful of confirmed cases where people had reason to do this, so we don't want to falsely give the impression that everyone should be doing it. I definitely don't think we should stir up fear about it by talking about firewalls and antivirus.